policies/optimized-images.md

```diff
@@ -39,60 +62,70 @@ Optimized image policies introduce a set of restrictions (policies) on images th
 
 On a web page, the number of pixels of a container determines the resolution of an image served inside. It is unnecessary to use an image that is much larger than what the viewing device can actually render; for example, serving a desktop image to mobile contexts, or serving an image intended for high-pixel-density screens to a low-pixel-density device. This results in unnecessary network traffic and downloaded bytes. `oversized-images` is a policy controlled feature that restricts images to be no more than X times bigger than the container size.
 
-When a document is disallowed to use `oversized-images` policy, its `<img>` elements that are more than X times larger than its container size will be rendered as a placeholder image.
-
+When a document disallows the `oversized-images` policy, the `<img>` elements that are more than X times larger than the container size will be replaced with placeholder images.
```

Comment:
We should probably say "more than X times larger than the container size in either dimension" to make it clear that we're looking at both width and height separately, and not just counting pixels.